backend/backend/api/routes.py

```diff
@@ -293,6 +295,11 @@ async def github_comment(repo, pr_number, request):
     return response
 
 
+async def github_pr(repo, pr_number):
+    response = await review_github_pr(Context(), repo, int(pr_number))
```

Comment:
Consider breaking down the `github_pr` function into smaller functions for better modularity and reusability.